### DISCUSSION TITLE
Want To Add a Text Box

### DISCUSSION THREAD

**Discussion by salil177** [mirceapiturca/Sections#6] [Q&A] [answered]
Can you please help me in creating a textbox with each result?

**Comment by mirceapiturca** [comment, marked as answer]
The quiz is designed to display a list of products based on the selections made. 

But you can try this:
1. Add your textbox in the div element with `quiz-main` class name
https://github.com/mirceapiturca/Sections/blob/fd60d0f6bb5f8bb40cc14dc81071ad25834c5012/Quiz/sections/quiz.liquid#L365
2. Comment this line `last(nodes.panels).querySelector('.quiz-main').innerHTML ` https://github.com/mirceapiturca/Sections/blob/fd60d0f6bb5f8bb40cc14dc81071ad25834c5012/Quiz/assets/quiz.js#L109

**Comment by salil177** [comment]
Step - 1 If I select a question 1 then a particular answer.
Step 2 - Then select question 2 with a answer.

For every combination want to display a different text.
Is this possible?

**Reply by mirceapiturca** [reply]
It would be possible but you need to modify the quiz behavior.

1. You would need to store somewhere the text per each combination. Could be in the section settings or a JSON metefield.
2. You will need to modify the function that shows the quiz results.
Have a look at https://github.com/mirceapiturca/Sections/blob/master/Quiz/assets/quiz.js#L32, this compose function is executed at the end of the quiz.
